jdi-light/src/main/java/com/epam/jdi/light/elements/composite/WebPage.java

```diff
@@ -50,45 +73,64 @@
     public String title = "";
 
     public String checkUrl;
-    public CheckTypes checkUrlType = CONTAINS;
-    public CheckTypes checkTitleType = NONE;
+  //  public CheckTypes checkUrlType = CONTAINS;
+  //  public CheckTypes checkTitleType = NONE;
 
     public <T> Form<T> asForm() {
-        return new Form<>().setPageObject(this).setup(Form.class,e->e.setName(getName()+" Form").setParent(this));
+        return new Form<>().setPageObject(this).setup(Form.class, e -> e.setName(getName() + " Form").setParent(this));
+    }
+
+    private static Safe<String> currentPageSingleton = null;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5327794)